bin/dummy_mentor

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
+questionnaire=$1
+default_page=index.html
+last_page=final.html
+questionnaire_page=questionnaire.html
+
+cd public/dummy
+
+rm -rf mentor
```

Comment:
Voordat we allemaal rm -rf dingen gaan aanroepen, kun je in dit script kijken of we wel in de goede directory zitten? Bijv dat pwd + Gemfile.lock bestaat oid? Misschien zelfs al voor die cd public/dummy

Reply:
done